README.md

```diff
@@ -14,57 +12,80 @@ SPDX-License-Identifier: Apache-2.0
+#### Linux and Mac
 
-#### Set up `dev-env` on Linux or macOS
+On Linux and Mac `dev-env` can be installed with:
 
-1. Use `cd daml` to switch into the new `daml` repository you just cloned
-2. Install Nix by running: `bash <(curl https://nixos.org/nix/install)`
-3. Enter `dev-env` by running: `eval "$(dev-env/bin/dade assist)"`
+1. Install Nix by running: `bash <(curl https://nixos.org/nix/install)`
+2. Enter `dev-env` by running: `eval "$(dev-env/bin/dade assist)"`
 
 If you don't want to enter `dev-env` manually each time using `eval "$(dev-env/bin/dade assist)"`,
 you can also install [direnv](https://direnv.net). This repo already provides a `.envrc`
 file, with an option to add more in a `.envrc.private` file.
 
-#### Set up `dev-env` on Windows
+#### Windows
+
+On Windows you need to enable long file paths by running the following command in an admin powershell:
+
+```
+Set-ItemProperty -Path 'HKLM:\SYSTEM\CurrentControlSet\Control\FileSystem' -Name LongPathsEnabled -Type DWord -Value 1
+```
+
+Then start `dev-env` from PowerShell with:
+
+```
+.\dev-env\windows\bin\dadew.ps1 install
+.\dev-env\windows\bin\dadew.ps1 sync
+.\dev-env\windows\bin\dadew.ps1 enable
+```
+
+In all new PowerShell processes started, you need to repeat the `enable` step.
 
-We're working on Windows support (for both end users and developers), but it's not ready yet.
-[Sign up](https://hub.daml.com/sdk/windows) to be notified when it is available.
+### 3. First build and test
 
-### 3. Build the source code
+We have a single script to build most targets and run the tests. On Linux and Mac run `./build.sh`. On Windows run `.\build.ps1`. Note that these scripts may take over an hour the first time.
 
-Run `bazel build //...`
+To just build do `bazel build //...`, and to just test do `bazel test //...`. To read more about Bazel and how to use it, see [the Bazel site](https://bazel.build).
 
-This builds the code, and will likely take an hour or more.
+### 4. Installing a local copy
 
-Now you've built, rebuilding the code after a change will be much faster because Bazel caches
-unchanged build artefacts. To read more about Bazel and how to use it, see [the Bazel site](https://bazel.build).
+On Linux and Mac run `da-sdk-head` which installs a version of the SDK with version number `0.0.0`. Set the `version:` field in any DAML project to 0.0.0 and it will use the locally installed one.
 
-To run the tests, run `bazel test //...`
+On Windows:
 
-### 4. Contribute!
+```
+bazel build //release:sdk-release-tarball
+tar -vxf .\bazel-genfiles\release\sdk-release-tarball.tar.z
+cd sdk-*
+daml\daml.exe install . --activate
+```
 
-If you are looking for ideas on how to contribute, please browse our
-[issues](https://github.com/digital-asset/daml/issues).
+That should tell you what to put in the path, something along the lines of `C:\Users\admin\AppData\Roaming\daml\bin`.
+Note that the Windows build is not yet fully functional.
```

Comment:
Can we make this a bit more visible, e.g., by moving it at the beginning of the `On Windows` section? I’d rather spend our time on fixing issues than on explaining the same issues over and over because people expect it to be fully functional.